### PR TITLE
style edit button

### PR DESCRIPTION
**Pull request by arsalan2243**
## Description
stylized the button edit for page "my profile". due to button looking too generic i decided to change it a bit and make it more interesting.

#### Video Link

https://www.loom.com/share/deaff77dd82d430fa36b89feb248a8cc

## Type of change

- [x] New feature (style change)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by maidiance — CHANGES_REQUESTED** [review]
Pretty good styling overall, but please see my inline comments.

File: src/components/pages/Profile/EditProfile.js [hunk 1]
```diff
@@ -35,10 +35,36 @@ function EditProfile({ userInfo }) {
...
+    alignItems: 'center',
+    backgroundImage:
+      'radial-gradient(100% 100% at 100% 0, #5adaff 0, #5468ff 100%)',
+    appearance: 'none',
+    border: 0,
+    borderRadius: '6px',
+    boxShadow:
+      'rgba(45, 35, 66, .4) 0 2px 4px,rgba(45, 35, 66, .3) 0 7px 13px -3px,rgba(58, 65, 111, .5) 0 -3px 0 inset',
+    boxSizing: 'border-box',
+    //   color: #fff;
```
Comment (jrosado184) [inline src/components/pages/Profile/EditProfile.js:47]: This commented out code should probably be removed

File: src/components/pages/Profile/EditProfile.js [hunk 2]
```diff
@@ -35,10 +35,36 @@ function EditProfile({ userInfo }) {
...
+    paddingRight: '16px',
+    position: 'relative',
+    textAlign: 'left',
+    textDecoration: 'none',
+    transition: 'box-shadow .15s,transform .15s',
+    userSelect: 'none',
+    touchAction: 'manipulation',
+    whiteSpace: 'nowrap',
+    willChange: 'box-shadow,transform',
+    fontSize: '18px',
```
[the remark this answers is not in the document]
Comment (arsalan2243) [reply src/components/pages/Profile/EditProfile.js:65]: ofcourse. 
what should i change it too?

[the remark this answers is not in the document]
Comment (maidiance) [reply src/components/pages/Profile/EditProfile.js:65]: 1.125rem, give or take. Feel free to adjust as you like :)

File: src/components/pages/Profile/EditProfile.js (hunk 1 above)
Comment (arsalan2243) [reply src/components/pages/Profile/EditProfile.js:47]: yes. That was my mistake. I realized it while shooting the video.
i will remove it on my next correction.

**Review by morgankj — APPROVED** [review]
Great suggestions Winnie! Good small change, great job keeping the files changed to 1. Nice job removing unnecessary comments. May need stakeholder approval, but good suggestion either way!

File: src/components/pages/Profile/EditProfile.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (arsalan2243) [reply src/components/pages/Profile/EditProfile.js]: ok i fixed it. please take a look.

File: src/components/pages/Profile/EditProfile.js [hunk 3]
```diff
@@ -35,10 +35,36 @@ function EditProfile({ userInfo }) {
...
+    appearance: 'none',
+    border: 0,
+    borderRadius: '6px',
+    boxShadow:
+      'rgba(45, 35, 66, .4) 0 2px 4px,rgba(45, 35, 66, .3) 0 7px 13px -3px,rgba(58, 65, 111, .5) 0 -3px 0 inset',
+    boxSizing: 'border-box',
+    //   color: #fff;
+    cursor: 'pointer',
+    display: 'inline-flex',
+    fontFamily: '"JetBrains Mono",monospace',
```
[the remark this answers is not in the document]
Comment (arsalan2243) [reply src/components/pages/Profile/EditProfile.js]: changed the font.

**Review by jcpcab — APPROVED** [review]
Good thinking to add style to the button! While having minimal changes to files.

**Review by hollberg — APPROVED** [review]
I really like the look of the button format. Great improvement!

**Review by BrittniKarat — COMMENTED** [review]
I love the way this button looks. I think the design is eye catching and stylish. My only concern would be the consistency with the rest of the website. Have we consulted the figma documentation on [design standards](https://www.figma.com/file/cwOoFddP7i0BdcP1oB7Kco/Underdog-Devs-Ant-Design?node-id=1%3A3) or submitted this design to Swapnali for approval?

**Review by lgallo95 — APPROVED** [review]
Code and design looks good. Good job with this easy win!

**Comment by paulstgermain** [comment]
@arsalan2243 I would highly recommend taking a look at @BrittniKarat's comment above here - we want to keep things consistent! If you can make adjustments for this, this PR would likely be good to merge up.

**Review by phillipgalbraith — APPROVED** [review]
Why are these styles in the .js?  It looks like we are moving styles to .css as a team.  This is a well scoped PR.  Good job resolving the concerns others have brought up.  It looks clean.

**Review by OrdainedRat — APPROVED** [review]

**Review by beefybroccoli — APPROVED** [review]
The scope is sufficiently small. The style looks very nice.

**Review by wscottreynders — APPROVED** [review]
Looks great. That button looks very interesting.

**Pull request merged**